src/commons/data.js

```diff
@@ -0,0 +1,8 @@
+const data = [
+  { id: 1, link: '/', title: 'Books' },
+  { id: 2, link: '/categories', title: 'Categories' },
```

Comment:
- Hey, the categories should be a component and it should display a button with the text `check status`. Please, see the image below to fix this part of the project requirement below:




![Screenshot from 2023-01-20 14-29-37](https://user-images.githubusercontent.com/59397159/213706909-cf97856a-c948-49f5-89b2-c0fb07766e02.png)